go.mod

```diff
@@ -59,3 +60,5 @@ replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+inc
 
 // Required by Helm
 replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309
+
+replace github.com/kiegroup/kogito-operator => /home/karel/src/github.com/kiegroup/kogito-cloud-operator
```

Comment:
yes, will remove it once https://github.com/kiegroup/kogito-operator/pull/806 is merged
then it would be possible to adjust and run BDD tests to make sure that they work

Reply:
Done